src/FakeItEasy.Analyzer/DiagnosticDefinitions.cs

```diff
@@ -0,0 +1,50 @@
+namespace FakeItEasy.Analyzer
+{
+    using System;
+    using System.Collections.Immutable;
+    using System.Reflection;
+    using Microsoft.CodeAnalysis;
+
+    internal static class DiagnosticDefinitions
+    {
+        public static DiagnosticDescriptor UnusedCallSpecification { get; } = CreateDiagnosticDescriptor(nameof(UnusedCallSpecification), "FakeItEasy0001", "Usage", DiagnosticSeverity.Error, true);

```

Comment:
Should "Usage" be "FakeItEasy.Usage"?

The docs for `DiagnosticDescriptor` say:

```C#
        //   category:
        //     The category of the diagnostic (like Design, Naming etc.). For example, for CA1001:
        //     "Microsoft.Design".
```